doc/builders/packages/firefox.section.md

```diff
@@ -38,3 +39,12 @@ The `wrapFirefox` function allows to pass policies, preferences and extension th
   };
 }
 ```
+
+If `enableExtraExtensions = true` then all manually installed addons will be uninstalled from your browser profile.
```

Comment:
This option sounds reverse to me. Enabling extra extensions sounds like it is in addition to those that I've already installed in my profile. Setting this to false sounds like it will only allow those that have been declared here.